src/ansys/math/core/math.py

```diff
@@ -1494,8 +1494,20 @@ def dot(self, vec) -> float:
         self._mapdl.run(f"*DOT,{self.id},{vec.id},py_val")
         return self._mapdl.scalar_param("py_val")
 
-    def asarray(self) -> np.ndarray:
-        """Return this vector as a NumPy array.
+    def asarray(self, dtype=None) -> np.ndarray:
+        """Return the vector as a NumPy array.
+
+        Parameters
+        ----------
+        dtype : numpy.dtype, optional
+            NumPy data type to upload the array as. The options are ``np.double``,
+            ``np.int32``, and ``np.int64``. The default is the current array
```

Comment:
Maybe you can link the numpy objetcts
```suggestion
            NumPy data type to upload the array as. The options are `np.double <numpy.double>`_,
            `np.int32 <numpy.int32>`_, and `np.int64 <numpy.int64>`_. The default is the current array
```